aiagents4pharma/talk2knowledgegraphs/tests/test_utils_enrichments_ollama.py

```diff
@@ -10,38 +10,17 @@
 def fixture_ollama_config():
     """Return a dictionary with Ollama configuration."""
     return {
-        "model_name": "gemma2:2b",
+        "model_name": "smollm2:360m",
         "prompt_enrichment": """
-            You are a helpful expert in biomedical knowledge graph analysis.
-            Your role is to enrich the inputs (nodes or relations) using textual description.
-            A node is represented as string, e.g., "ADAM17" in the input list, while a relation is
-            represented as tuples, e.g., "(NOD2, gene causation disease, Crohn disease)".
-            All provided information about the node or relations should be concise
-            (a single sentence), informative, factual, and relevant in the biomedical domain.
+            Given the input as a list of strings, please return the list of addditional information of
+            each input terms using your prior knowledge.
 
-            ! IMPORTANT ! Make sure that the output is in valid format and can be parsed as
-            a list of dictionaries correctly and without any prepend information.
-            DO NOT forget to close the brackets properly.
-            KEEP the order consistent between the input and the output.
-            See <example> for reference.
-
-            <example>
-            Input: ["ADAM17", "IL23R"]
-            Output: [{{"desc" : "ADAM17 is a metalloproteinase involved in the shedding of
-            membrane proteins and plays a role in inflammatory processes."}}, {{"desc":
-            "IL23R is a receptor for interleukin-23, which is involved in inflammatory responses
-            and has been linked to inflammatory bowel disease."}}]
-            </example>
-
-            <example>
-            Input: ["(NOD2, gene causation disease, Crohn disease)", "(IL23R,
-            gene causation disease, Inflammatory Bowel Disease)"]
-            Output: [{{"desc" : "NOD2 is a gene that contributes to immune responses and has
-            been implicated in Crohn's disease, particularly through genetic mutations linked to
-            inflammation."}}, {{"desc" : "IL23R is a receptor gene that plays a role in
-            immune system regulation and has been associated with susceptibility to
-            Inflammatory Bowel Disease."}}]
-            </example>
+            Example:
+            Input: ['acetaminophen', 'aspirin']
+            Ouput: ['acetaminophen is a medication used to treat pain and fever', 
+            'aspirin is a medication used to treat pain, fever, and inflammation']
+            
+            Do not include any pretext as the output, only the list of strings enriched.
```

Comment:
Maybe you can also include a few 🔨 and 🔫 emojis in the prompt to really motivate it to follow directions 😉.